src/components/ListItem.js

```diff
@@ -21,11 +21,7 @@ const ListItem = ({ item, isChild }) => {
     e.stopPropagation();
     e.preventDefault();
 
-    console.log('playerState', playerState);
-    const href = e.target.href;
-
-    // TODO: Calculate the time here from the URL
-    playerState.media.setCurrentTime(120);
+    dispatch({ clickedUrl: e.target.href, type: 'seekPlayer' });
```

Comment:
What about maybe calling this action type `navItemClicked`?   I know eventually it will look into something with the player, but this immediate action semantically seems like it just relates to a navigation click.

Reply:
Yeah, that sounds like a better name for this. I will change it :+1: